__tests__/get-articles-api.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+
+const db = require("../db/connection");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe("GET /api/articles", () => {
+  test("that responds with all the articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {

```

Comment:
need to also check for length here incase you get any empty array - otherwise this will pass accidentally.